docs/getting-started/install-endpoint.asciidoc

```diff
@@ -26,46 +26,56 @@ image::images/install-endpoint/security-integration.png[Search result for "Endpo
 . To complete the integration, continue to the next section to install the {agent} on your hosts.
 
 [discrete]
 [[enroll-security-agent]]
-== Configure and enroll the Elastic Agent
+== Configure and enroll the {agent}
 
 To configure the {agent}, Endpoint Security requires enrollment through Fleet to enable the integration.
 
-IMPORTANT: Before you add an {agent}, a {fleet-server} must be running. See {fleet-guide}/add-a-fleet-server.html[Add a {fleet-server}].
+[IMPORTANT]
+=====
+Before you add an {agent}, a {fleet-server} must be running. Refer to {fleet-guide}/add-a-fleet-server.html[Add a {fleet-server}].
+
+Endpoint Security cannot be integrated with an {agent} in Standalone mode.
+=====
 
 [discrete]
 [[fleet-server-upgrade]]
-=== Important information about the new {fleet-server}
+=== Important information about {fleet-server}
 
 NOTE: If you are running an {stack} version earlier than 7.13.0, you can skip this section.
 
-If you have upgraded to an {stack} version that includes the new {fleet-server} (>=7.13.0), you will need to redeploy your agents. Review the following scenarios to ensure you take the appropriate steps.
+If you have upgraded to an {stack} version that includes {fleet-server} (>=7.13.0), you will need to redeploy your agents. Review the following scenarios to ensure you take the appropriate steps.
 
 * If you redeploy the {agent} to the same machine through the {fleet} application after you upgrade, a new agent will appear.
-* If you want to remove the {agent} entirely without transitioning to the {fleet-server}, then you will need to manually uninstall the agent on the machine. This will also uninstall the endpoint. See {fleet-guide}/uninstall-elastic-agent.html[Uninstall Elastic Agent].
-* In the rare event that the {agent} fails to uninstall, you might need to manually uninstall the endpoint. See <<uninstall-endpoint, Uninstall an endpoint>> at the end of this topic.
+* If you want to remove the {agent} entirely without transitioning to the {fleet-server}, then you will need to manually uninstall the agent on the machine. This will also uninstall the endpoint. Refer to {fleet-guide}/uninstall-elastic-agent.html[Uninstall Elastic Agent].
+* In the rare event that the {agent} fails to uninstall, you might need to manually uninstall the endpoint. Refer to <<uninstall-endpoint, Uninstall an endpoint>> at the end of this topic.
 
-NOTE: Endpoint Security cannot be integrated with an {agent} in Standalone mode.
+[discrete]
+[[enroll-agent]]
+=== Add the {agent}
 
 . Go to *{fleet}* -> *Agents* -> **Add agent**.
 +
 [role="screenshot"]
 image::images/install-endpoint/add-agent.png[Add agent flyout on the Fleet page.]
+
+. Select an agent policy for the {agent}. You can select an existing custom policy, or select **Create new agent policy** to create a new one. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
```

Comment:
```suggestion
. Select an {agent} policy for the {agent}. You can select an existing custom policy, or select **Create new agent policy** to create a new one. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
```

Reply:
Again, ignore rec if it makes the sentence too verbose.

Reply:
I think in this case "agent policy" is a preferred term, as it's used in the UI.